content/Software and Downloads/IDE Settings/Find-sketches-libraries-boards-cores-and-other-files-on-your-computer.md

```diff
@@ -0,0 +1,150 @@
+
+Example sketches can be found in the _File > Examples_ menu.
+
+![](img/ide-menu-file-examples.png)
+
+* **Built-in Examples:** Examples packaged with Arduino IDE. They are located inside the application folder.
+  * In **Arduino IDE 2.0** they are stored in the `resources/app/node_modules/arduino-ide-extension/Examples` subfolder.
+  * In **Arduino IDE 1.8** they are stored in the `libraries` subfolder for Windows and Linux, and `Contents/Java/libraries` on macOS.
+* **Examples for \<board name\>:** Stored inside the board platform's `libraries` subfolder, e.g. `Arduino15/packages/arduino/hardware/avr/1.8.4/libraries/Wire`. These are only accessible by Arduino IDE when one of the boards from that package is selected from the _Tools > Board_ menu.
+* **Examples from Custom Libraries:** Installed with the Library Manager or the _Sketch > Include Library > Add. ZIP Library_ dialog.
+* **Examples for any board:** Installed with the Library Manager or the _Sketch > Include Library > Add. ZIP Library_ dialog. <!-- 1.x only? -->
+
+When an Example is open, you can select _Sketch > Show Sketch Folder_ to reveal the location in your system's file manager.
+
+---
+
+<a id="libraries"></a>
+
+## Libraries
+
+**Libraries installed with the Library Manager** are stored inside a Sketchbook subfolder:
+
+1. Open the Sketchbook folder (detailed instructions [here](https://support.arduino.cc/hc/en-us/articles/4412950938514)).
+   * **Windows:** `C:\Users\{username}\Documents\Arduino`
+   * **macOS:** `/Users/{username}/Documents/Arduino`
+   * **Linux:** `/home/{username}/Arduino`
+
+   > For detailed instructions, see [Open the Sketchbook](https://support.arduino.cc/hc/en-us/articles/4412950938514-Open-the-Sketchbook).
+
+2. Open the `libraries` subfolder.
+
+3. The files for each library are stored within a folder with the library name:
+
+   ```
+   Arduino
+   └── libraries
+        └── ArduinoBLE
+            ├── CHANGELOG
+            ├── LICENSE
+            ├── README.md
+            ├── examples
+            ├── extras
+            ├── keywords.txt
+            ├── library.properties
+            └── src
+    ```
+
+**Libraries included in board packages** are stored within the board package folder's `libraries` subfolder, see [Board platforms and cores](#boards).
+
+<!-- To add or remove libraries, use the [Library Manager](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries). -->
+
+> **An easy way to find a library with examples:**
+>
+> 1. Open _File > Examples_ in the menu bar.
+> 2. Put your cursor over the name of the library you want to find.
+> 3. Select any of the library examples.
+> 4. Select _Sketch > Show Sketch Folder_ from the menu bar.
+
+---
+
+<a id="boards"></a>
+
+## Board platforms and cores
+
+Board platforms installed with the Board Manager are stored inside the Arduino15 folder.
+
+1. Open the Arduino15 folder (detailed instructions [here](https://support.arduino.cc/hc/en-us/articles/360018448279)).
```

Comment:
Looks more concise like this IMO, since there is a link to the same page in line 96
```suggestion
1. Open the Arduino15 folder.
```